src/ngx_http_lua_shdict.c

```diff
@@ -43,13 +51,20 @@ static ngx_inline ngx_shm_zone_t *ngx_http_lua_shdict_get_zone(lua_State *L,
 #define NGX_HTTP_LUA_SHDICT_ADD         0x0001
 #define NGX_HTTP_LUA_SHDICT_REPLACE     0x0002
 #define NGX_HTTP_LUA_SHDICT_SAFE_STORE  0x0004
+#define NGX_HTTP_LUA_SHDICT_LEFT        0x0001
```

Comment:
Better put a blank line above this line since these 2 constants do not belong to the previous ones.